src/main/java/io/vertx/core/net/NetServerOptions.java

```diff
@@ -531,6 +554,62 @@ public NetServerOptions setTrafficShapingOptions(TrafficShapingOptions trafficSh
     return this;
   }
 
+  /**
+   * @return the time in seconds the connection needs to remain idle before TCP starts sending keepalive probes
+   */
+  public int getTcpKeepAliveIdleSeconds() {
```

Comment:
why not make this part of TCPSSLOptions instead ? it does not seem specific to TCP server

Reply:
Done